.github/workflows/e2e.yaml

```diff
@@ -59,6 +56,12 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.17.x
+      - name: Run tests
+        run: |
+          mkdir tmp-download; cd tmp-download; go mod init go-download;
+          GOBIN="${GITHUB_WORKSPACE}/build/gobin" go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+          cd ..; rm -rf tmp-download
```

Comment:
Our `arm64` runners don't seem to be able to deal with Makefile's `go-install-tool` when installing `setup-envtest`. Although it works fine for all other targets.